src/models/secret.ts

```diff
@@ -13,15 +13,18 @@ export class Secret {
   public isParanoia: boolean
   public hasSocialRecovery: boolean
   public interactionSetting: InteractionSetting
+  public passphrase: string
 
   public wallets: AirGapWallet[]
 
   private twofactor: string
+  
 
-  constructor(seed: string, label: string = '', isParanoia = false, interactionSetting = InteractionSetting.UNDETERMINED) {
+  constructor(seed: string, passphrase: string = '', label: string = '', isParanoia = false, interactionSetting = InteractionSetting.UNDETERMINED) {
```

Comment:
We have to be careful here, because the "label" might be used as the passphrase if we don't change all the Secret constructors.

Reply:
I think I get it. So even though these are named parameters one in TS can just pass arguments? Then I guess the passphrase should go to the end of the constructor. Is this what you meant? I guess I got ahead of myself and should have just added it as the last argument as a safety rule of thumb. Good point.

Reply:
Yes, the problem is that TS just checks the types, but because the label and the passphrase are both strings, the compiler will not complain and it will simply accept the label as the passphrase, so to speak. We can probably move it one to the right (after label) because `isParanoia` is a boolean and it would complain if the call signatures don't match.